cwms-data-api/src/test/java/cwms/cda/api/MeasurementControllerTestIT.java

```diff
@@ -0,0 +1,654 @@
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import static org.hamcrest.Matchers.*;
+
+@Tag("integration")
+final class MeasurementControllerTestIT extends DataApiTestIT {
+
+    private static final String OFFICE_ID = TestAccounts.KeyUser.SPK_NORMAL.getOperatingOffice();
+    private static final List<Stream> STREAMS_CREATED = new ArrayList<>();
+    private static final List<String> STREAM_LOC_IDS = new ArrayList<>();
+
+    @BeforeAll
+    public static void setup() throws SQLException {
+        String testLoc = "StreamLoc321"; // match the stream location name in the json file
+        createLocation(testLoc, true, OFFICE_ID, "STREAM_LOCATION");
+        STREAM_LOC_IDS.add(testLoc);
+        createAndStoreTestStream("ImOnThisStream2");
+    }
+
+    static void createAndStoreTestStream(String testLoc) throws SQLException {
+        createLocation(testLoc, true, OFFICE_ID, "STREAM");
+        CwmsDatabaseContainer<?> db = CwmsDataApiSetupCallback.getDatabaseLink();
+        db.connection(c -> {
+            StreamDao streamDao = new StreamDao(getDslContext(c, OFFICE_ID));
+            Stream streamToStore = new Stream.Builder()
+                    .withId(new CwmsId.Builder()
+                            .withOfficeId(OFFICE_ID)
+                            .withName(testLoc)
+                            .build())
+                    .withLength(100.0)
+                    .withLengthUnits("km")
+                    .build();
+            STREAMS_CREATED.add(streamToStore);
+            streamDao.storeStream(streamToStore, false);
+        }, CwmsDataApiSetupCallback.getWebUser());
+    }
+
+    @AfterAll
+    public static void tearDown() {
+        for (Stream stream : STREAMS_CREATED) {
+            try {
+                CwmsDatabaseContainer<?> db = CwmsDataApiSetupCallback.getDatabaseLink();
+                db.connection(c -> {
+                    StreamDao streamDao = new StreamDao(getDslContext(c, OFFICE_ID));
+                    try {
+                        streamDao.deleteStream(stream.getId().getOfficeId(), stream.getId().getName(), DeleteRule.DELETE_ALL);
+                    } catch (Exception e) {
+                        // ignore
+                    }
+                }, CwmsDataApiSetupCallback.getWebUser());
+            } catch (SQLException ex) {
+                throw new RuntimeException(ex);
+            }
+        }
+        STREAMS_CREATED.clear();
+        for(String measLoc: STREAM_LOC_IDS)
+        {
+            try {
+                CwmsDatabaseContainer<?> db = CwmsDataApiSetupCallback.getDatabaseLink();
+                db.connection(c -> {
+                    MeasurementDao measDao = new MeasurementDao(getDslContext(c, OFFICE_ID));
+                    try {
+                        measDao.deleteMeasurements(OFFICE_ID, measLoc, null, null, null, null, null, null, null, null, null, null, null);
+                    } catch (Exception e) {
+                        // ignore
```

Comment:
Same comment as above

Reply:
I added a logging at info level to display a failure, though it is expected that this will fail if the deletion works as expected in the places where deletion is being tested.